Week2/js-exercises/oddOnesOut.js

```diff
@@ -0,0 +1,24 @@
+
+const myNumbers = [1, 2, 3, 4];
+let differentNumbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14]
+
+function doubleNumbers(number) {
+    return number * 2
+}
+
+function evenNumbers(number) {
+    return number % 2 == 0
+}
+
+//the following function requires an array as an input.
+
+function doubleEvenNumbers(numbers) {
+    let filteredNumbers = numbers.filter(evenNumbers)
+    let doubledNumbers = filteredNumbers.map(doubleNumbers)
```

Comment:
This is absolute correct! 
_Did you know that you can chain the array function calls?_
```javascript
let doubledNumbers = numbers.filter(evenNumbers).map(doubleNumbers)
```
**Extra challenge**
Can you do it with a single iteration of the array using reduce?